packages/@atjson/offset-annotations/src/annotations/instagram-embed.ts

```diff
@@ -1,6 +1,6 @@
 import { IframeEmbed } from "./iframe-embed";
 
-export class InstagramEmbed extends IframeEmbed {
+export class InstagramEmbed extends IframeEmbed<{ captioned?: boolean }> {
```

Comment:
Can we bit a more descriptive on this since it is specifically for `instagram captions` - `captioned` seems a bit vague since we have captions elsewhere